packages/backend/src/projectAdapters/dbtGitlabProjectAdapter.ts

```diff
@@ -37,7 +37,7 @@ export class DbtGitlabProjectAdapter extends DbtGitProjectAdapter {
         cachedWarehouse,
         dbtVersion,
     }: DbtGitlabProjectAdapterArgs) {
-        const remoteRepositoryUrl = `https://:${gitlabPersonalAccessToken}@${
+        const remoteRepositoryUrl = `https://lightdash:${gitlabPersonalAccessToken}@${
```

Comment:
This was another issue, username can't be empty 

https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#clone-repository-using-personal-access-token

Reply:
sorry, don't get this part. Does it have to be `lightdash`?

Reply:
according to the docs, it can be anything, as long as it is not empty, I tested this with lightdash and `oauth2` , both work. I think we use lightdash on github already